rust-toolchain

```diff
@@ -1 +1 @@
-stable
\ No newline at end of file
+stable
```

Comment:
What we actually want to do is make sure client library works on stable rust. Didn't know this could prevent users of this library to use beta or nightly. I'm wondering what could be best practice to fulfill this?

Reply:
As far as I am aware this is just an overridable "Default to this unless configured" option.

Reply:
Yeah, it only matters to people building this repo directly. My mistake. It's fine.